test/integration/simulations_controller_integration_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'minitest/autorun'
+require 'test_helper'
+require 'mocha/test_unit'
+
+require 'db_helper'
+require 'controller_integration_test_helper'
+
+class SimulationsControllerIntegrationTest < ActionDispatch::IntegrationTest
+  include DBHelper
+  include ControllerIntegrationTestHelper
+
+  def setup
+    super
+    authenticate_session!
+  end
+
+  def teardown
+    super
+  end
+
+  test 'get simulation scenarios' do
```

Comment:
Opis testu zmienić na: "simulations GET index should return simulation scenarios id accessible by user"